packages/fast-components-react-msft/src/action-toggle/action-toggle.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Action toggle",
+    "description": "An action toggle component's schema definition.",
+    "type": "object",
+    "id": "action-toggle",
+    "properties": {
+        "disabled": {
+            "title": "Disabled",
+            "type": "boolean"
+        },
+        "appearance": {
+            "title": "Appearance",
+            "type": "string",
+            "default": "primary",
+            "enum": [
+                "primary",
+                "justified",
+                "lightweight",
+                "outline"
+            ]
+        },
+        "selected": {
+            "title": "Selected",
+            "type": "boolean"
+        },
+        "selectedText": {
+            "title": "Selected text",
+            "type": "string"
+        },
+        "unselectedText": {
+            "title": "Unselected text",
+            "type": "string"
+        },
+        "selectedARIALabel": {
```

Comment:
I would just remove `ARIA` all together from the prop names. If we left it in it would be `Aria` but I think we should remove

Reply:
So there are two text fields for "selected" - the one that populates the actual label and the one used for accessibility.  We still need the accessibility string if we have a glyph only version of this component.  How should we differentiate between the two name wise?

Reply:
I'm not sure about this one.  Feels like there are specific props for accessibility and calling this out could be useful.  Otherwise, context could be lost for "selectedLabel" vs "selectedARIALabel". Also, this can be left in as "ARIA" if we think it's helpful since I would consider it a well known industry acronym.

Reply:
In the past we have just labeled it like `selectedLabel` - we have never used Aria. So we would have `text` and `label`, and it was inferred that `text` was visual and `label` was not. 

I could see this being confusing so I am open to renaming BUT we have other props in the library that deal with Aria that are _not_ labeled this way, and that in turn is confusing.

Reply:
We should align with the current props we have in existence for this because we talked over those for hours.

In my opinion we can document what and when this is useful for. For instance, the only reason this existed in previous implementations was when there was just a glyph. Because we have selected/unselected, we need to enumerate this, but it should really *only* be used if there is just a glyph.

Reply:
We ended up going with "label" for the accessibility stuff and "content" for the regular content (which is now a react node and not just a string).